### PR TITLE
Challenge/isabel gomez tour of heroes react router

### PR DESCRIPTION
**Pull request by IsabelGomez87**

### CODE REVIEW

File: Isabel-Gomez/challenge/tour-of-heroes-react/src/components/common/Anchorates.jsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
-const Anchorates = ({ text, href }) => (
-  <a href={href}>{text}</a>
+const Anchorates = ({ text, to = 'null' }) => (
+  <Link to={to}>{text}</Link>
 );
 
 export default Anchorates;
```
Comment (edalandete) [inline Isabel-Gomez/challenge/tour-of-heroes-react/src/components/common/Anchorates.jsx:8]: A component for the anchors!! Nice job!!

File: Isabel-Gomez/challenge/tour-of-heroes-react/src/components/common/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const Card = ({ data }) => {
+  const [showDetails, setShowDetails] = useState(false);
+
+  return (
+    <>
+      <li>
+        <button type="button" onClick={() => setShowDetails(!showDetails)}>+</button>
+        {data.name}
+      </li>
+      {showDetails && <Details data={data} />}
```
Comment (edalandete) [inline Isabel-Gomez/challenge/tour-of-heroes-react/src/components/common/Card.jsx:13]: I think you can put the ` {showDetails && <Details data={data} />}` inside the li

File: Isabel-Gomez/challenge/tour-of-heroes-react/src/data/heroes.js [hunk 3]
```diff
@@ -1,14 +1,164 @@
...
-  { id: 14, name: 'Celeritas' },
-  { id: 15, name: 'Magneta' },
-  { id: 16, name: 'RubberMan' },
-  { id: 17, name: 'Dynama' },
-  { id: 18, name: 'Dr IQ' },
-  { id: 19, name: 'Magma' },
-  { id: 20, name: 'Tornado' }
+  {
+    id: 'dc-batman',
+    name: 'Batman',
```
Comment (edalandete) [inline Isabel-Gomez/challenge/tour-of-heroes-react/src/data/heroes.js:4]: ❤️

**Review by edalandete — APPROVED** [review]
Amaizing job

**Review by cristian-bars — APPROVED** [review]

**Review by davzlum — APPROVED** [review]
Buen trabajo Isabel!

**Review by IrenePerezAlonso — APPROVED** [review]
Está genial!!

**Review by Laura-SM — APPROVED** [review]
Good job!

**Review by gteixi — APPROVED** [review]

**Pull request merged**